src/popup/components/form/FormAccountInput.vue

```diff
@@ -0,0 +1,265 @@
+    const selectedAddress = ref<IAddressBookEntry | undefined>();
+    const filteredOptions = computed(() => (
+      addressBookFiltered.value.filter((entry: IAddressBookEntry) => (
+        entry.address.toLowerCase().includes(props.modelValue.toString().toLowerCase())
+            || entry.name.toLowerCase().includes(props.modelValue.toString().toLowerCase())
+      ))
+    ));
+
+    function clearAddress() {
+      selectedAddress.value = undefined;
+      emit('update:modelValue', '');
+    }
+
+    function selectValue(entry: IAddressBookEntry, $event: Event) {
+      $event.preventDefault();
+      emit('update:modelValue', entry?.address);
+      isDropdownOpen.value = false;
+    }
+
+    onMounted(() => {
+      watch(() => props.modelValue, () => {
+        const searchAddress = addressBookFiltered.value
+          ?.find((entry: IAddressBookEntry) => (entry.address === props.modelValue));
+        if (searchAddress) {
+          selectedAddress.value = searchAddress;
+        }
+        if (!props.modelValue) { // If it's cleared externally
+          selectedAddress.value = undefined;
+        }
+
+        isDropdownOpen.value = props.modelValue.toString().length >= 2
+            && filteredOptions.value.length > 0;
+        if (addressBookFiltered.value?.find(
+          (entry) => (entry.address === props.modelValue.toString()),
+        )) {
+          isDropdownOpen.value = false;
+        }
+      }, { immediate: true });
+    });
+
+    return {
+      filteredOptions,
+      selectedAddress,
+      clearAddress,
+      selectValue,
+      isDropdownOpen,
+      CircleCloseIcon,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '@/styles/variables' as *;
+@use '@/styles/typography';
+
+.close-icon {
+  padding: 0;
+  opacity: 0.5;
+}
+
+.truncated-text {
+  color: $color-white;
+}
+
+.input-field {
+  &:where(
+    &:has(input:focus),
+    &:has(textarea:focus),
+    &:has(label:active)
+  ) {
+    .dropdown-wrapper {
+      display: block;
+      top: calc(100% + 4px);
+      opacity: 1;
+      user-select: all;
+      pointer-events: all;
+    }
+  }
+
+  .dropdown-wrapper {
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    display: block;
+    background-color: $color-black;
+    border: none;
+    border-radius: $border-radius-interactive;
+    box-shadow: inset 0 0 0 2px rgba(255, 255, 255, 0.15), 2px 2px 8px 0px rgba(0, 0, 0, 0.33);
+    transition: 100ms ease-in-out;
+    max-height: 240px;
+    overflow: scroll;
+    opacity: 0;
+    user-select: none;
+    pointer-events: none;
+    padding: 5px 2px;
+    overflow-x: hidden;
+    overflow-y: auto;
+    z-index: -1;
+
+    .dropdown-item {
+      padding: 8px;
+      cursor: pointer;
+      user-select: none;
+      display: flex;
+      gap: 6px;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .dropdown-item:not(:last-child) {
+      border-bottom: 1px solid rgba(255, 255, 255, 0.15);
```

Comment:
Please use color variables instead of RGB values:
```suggestion
      border-bottom: 1px solid rgba($color-white, 0.15);
```